lib/lveditqmljs/src/codeqmlhandler.h

```diff
@@ -76,6 +76,7 @@ class LV_EDITQMLJS_EXPORT CodeQmlHandler : public QObject{
     );
     void setDocument(ProjectDocument* document);
     void rehighlightBlock(const QTextBlock& block);
+    // QPair<int, int> contextBlock(int position) Q_DECL_OVERRIDE;
```

Comment:
`QPair<int, int> contextBlock(int position)` was deleted in version 1.6.1 in favour of `QJSValue contextBlockRange(int cursorPosition);`